bind/gencffi_cdef.go

```diff
@@ -116,6 +217,8 @@ func (g *cffiGen) genCdefStructMemberGetter(s Struct, i int, f types.Object) {
 	}
 }
 
+// genCdefStructMemberSetter generates
+// C defintion of Setter for Go struct.
```

Comment:
`s/for Go/for a Go/`.
`s/defintion/definition/`
also: please everything on one line.